llvm_jni/main.bal

```diff
@@ -0,0 +1,13 @@
+import llvm_jni.llvm;
+import  ballerina/io;
+public function main() {
+    llvm:Context context = new;
+    llvm:Builder builder = context.createBuilder();
+    llvm:Module m = context.createModule();
+    llvm:Function mainFunction = m.addFunction("main", {returnType: "void", paramTypes: ["i64", "i1"]});
+    llvm:BasicBlock initBlock = mainFunction.appendBasicBlock();
+    builder.positionAtEnd(initBlock);
+    builder.ret(llvm:constInt("i64",0));
+    io:println(m.toString());
+    checkpanic m.toFile("test.ll");
```

Comment:
Our current Module has  `toString` and `writeFile` methods. We should probably change these to `printModuleToString` and `printModuleToFile` to better match the LLVM C API.

Reply:
Fixed